rust-query-crlite/src/main.rs

```diff
@@ -537,6 +537,7 @@ impl Intermediates {
                 if cert.verify_signature(Some(&issuer_spki)).is_ok() {
                     return Some(issuer_spki);
                 }
+                debug!("Not issued by {}", issuer.tbs_certificate.subject);
```

Comment:
Did you still want this debugging statement here?

Reply:
eh, I guess it's not that useful. I removed it.